apps/search/js/private_browsing.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+/* global newtab */
+
+(function(exports) {
+
+  function PrivateBrowsing() {
+    this.trigger = document.getElementById('private-window');
+    this.trigger.addEventListener('click', this);
+  }
+
+  PrivateBrowsing.prototype = {
+
+    /**
+     * General event handler.
+     * Currently dispatches a request to open a new private window
+     * from the newtab object.
+     */
+    handleEvent: function(e) {
+      newtab.requestPrivateWindow();
```

Comment:
Yeah, this file was more complex before, but I agree. Let's move it to newtab.js for now and we can revisit if needed.